charts/kubernetes-dashboard-proxy/Chart.yaml

```
@@ -1,5 +1,5 @@
 apiVersion: v1
 description: A reverse proxy with OpenID Connect for Kubernetes Dashboard
 name: kubernetes-dashboard-proxy
-version: 1.5.0
+version: 1.5.2
```

Please bump the minor version as follows:

```suggestion
version: 1.6.0
```

---

@int128 done